app/_hub/kong-inc/datadog-tracing/versions.yml

```diff
@@ -0,0 +1,7 @@
+strategy: gateway
+
+releases:
+  minimum_version: '3.2.x'
+
+overrides:
+  3.2.x: 0.1.0
```

Comment:
Why do we need an override for this plugin version? As far as I know, the plugin version _should_ match the Kong version starting in 3.0. If it doesn't, then something's wrong.